package.json

```diff
@@ -94,7 +94,7 @@
   },
   "scripts": {
     "start": "react-app-rewired start",
-    "build": "rollup --config && yarn createTsDec",
+    "build": "rollup --config && yarn tsc",
```

Comment:
Maybe in this case, it's worth just using the rollup typescript plugin instead?

Reply:
good point

Reply:
I have been unable to get it to work locally :/

Reply:
We can update later